src/index.ts

```diff
@@ -181,6 +181,10 @@ export class LinkChecker extends EventEmitter {
       let res = await gaxios.request<string>({
         method: opts.crawl ? 'GET' : 'HEAD',
         url: opts.url.href,
+        headers: {
```

Comment:
Let's define this once, globally, and then use it in both places where we need it.